src/lib/mdns/Advertiser.h

```diff
@@ -101,23 +107,39 @@ class BaseAdvertisingParams
     }
     const chip::ByteSpan GetMac() const { return chip::ByteSpan(mMacStorage, mMacLength); }
 
+    // Common Flags
+    Derived & SetMRPRetryIntervals(Optional<uint32_t> intervalIdle, Optional<uint32_t> intervalActive)
```

Comment:
nit: I think if those were handled by two separate setters, the setters wouldn't need to take `Optional` (they could take raw `uint32_t`).

Reply:
They do, in order to match the spec. The spec indicates that these fields can be omitted. If these are set to Optional() (ie, no value), they will not be added to the TXT record.

Reply:
@turon - do you know how these should be set? What should the idle default be?